sim/core/item_swaps.go

```diff
@@ -268,9 +268,19 @@ func (swap *ItemSwap) GetUnequippedItemBySlot(slot proto.ItemSlot) *Item {
 }
 
 func (swap *ItemSwap) ItemExistsInMainEquip(itemID int32, possibleSlots []proto.ItemSlot) bool {
+	if !swap.IsEnabled() {
+		return swap.character.Equipment.containsItemInSlots(itemID, possibleSlots)
+	}
 	return swap.originalEquip.containsItemInSlots(itemID, possibleSlots)
 }
 
+func (swap *ItemSwap) EnchantExistsInMainEquip(effectID int32, possibleSlots []proto.ItemSlot) bool {
+	if !swap.IsEnabled() {
+		return swap.character.Equipment.containsEnchantInSlots(effectID, possibleSlots)
+	}
```

Comment:
You will also need similar branching logic to this within `EligibleSlotsForEffect()`, or else the `possibleSlots` input here will always be empty if item swapping is not enabled.